config/opal_setup_cc.m4

```diff
@@ -41,14 +140,25 @@ AC_DEFUN([OPAL_SETUP_CC],[
     WRAPPER_CC="$CC"
     AC_SUBST([WRAPPER_CC])
 
-    # From Open MPI 1.7 on we require a C99 compiant compiler
-    AC_PROG_CC_C99
-    # The result of AC_PROG_CC_C99 is stored in ac_cv_prog_cc_c99
-    if test "x$ac_cv_prog_cc_c99" = xno ; then
-        AC_MSG_WARN([Open MPI requires a C99 compiler])
-        AC_MSG_ERROR([Aborting.])
-    fi
+    OPAL_PROG_CC_C11
```

Comment:
Is it *guaranteed* that if `$opal_cv_c11_supported==yes`, you won't need to call `AC_PROG_CC_C99`?

Reply:
Yes. If c11 support is detected then c99 support must exist as well. So there is no need to check c99.